### PR TITLE
[IMP] test_themes, theme_buzzy: s_form adaptations

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adapts the s_form customizations since this snippet has been redesigned in PR [1].

task-4048101
Part of task-3619705

[1]: https://github.com/odoo/odoo/pull/172763

Requires: 
- https://github.com/odoo/odoo/pull/172763

### CODE REVIEW

File: theme_buzzy/views/new_page_template.xml [hunk 1]
```diff
@@ -118,12 +118,6 @@
     </xpath>
 </template>
 
-<template id="new_page_template_s_website_form" inherit_id="website.new_page_template_s_website_form">
-    <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc2" separator=" "/>
-    </xpath>
-</template>
-
```
Comment (stefanorigano) [inline theme_buzzy/views/new_page_template.xml:126]: Let's not remove templates for now ;)

File: test_themes/tests/test_new_page_templates.py [hunk 2]
```diff
@@ -89,7 +89,7 @@
         's_product_catalog_dish_name', 's_product_catalog_dish_dot_leaders',
         's_table_of_content_vertical_navbar', 's_table_of_content_navbar_sticky', 's_table_of_content_navbar_wrap',
         's_timeline_card',
-        's_website_form_custom', 's_website_form_dnone', 's_website_form_field', 's_website_form_input', 's_website_form_mark',
+        's_website_form_custom', 's_website_form_dnone', 's_website_form_field', 's_website_form_input', 's_website_form_mark', 's_website_form_submit', 's_website_form_no_submit_label',
```
Comment (stefanorigano) [inline test_themes/tests/test_new_page_templates.py:92]: Not sure about this... makes sense to me but I'll leave it for the tech. review 😉

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Thanks @Brieuc-brd  👍

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋 
Branch updated, what do you think ?

**Review by stefanorigano — APPROVED** [review]
LGTM, thanks 👍

**Comment by sobo-odoo** [comment]
@robodoo delegate=robinlej

**Comment by robinlej** [comment]
@robodoo r+

**Pull request closed without merging**